docs/guides/experimental-exporter.md

```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
+| [databricks_ip_access_list](docs/resources/ip_access_list.md) | Yes |
+| [databricks_job](docs/resources/job.md) | Yes |
+| [databricks_library](docs/resources/library.md) | No |
+| [databricks_metastore](docs/resources/metastore.md) | No |
+| [databricks_metastore_assignment](docs/resources/metastore_assignment.md) | No |
+| [databricks_metastore_data_access](docs/resources/metastore_data_access.md) | No |
+| [databricks_mlflow_model](docs/resources/mlflow_model.md) | No |
+| [databricks_mlflow_experiment](docs/resources/mlflow_experiment.md) | No |
+| [databricks_mws_credentials](docs/resources/mws_credentials.md) | No |
+| [databricks_mws_customer_managed_keys](docs/resources/mws_customer_managed_keys.md) | No |
+| [databricks_mws_log_delivery](docs/resources/mws_log_delivery.md) | No |
+| [databricks_mws_networks](docs/resources/mws_networks.md) | No |
+| [databricks_mws_storage_configurations](docs/resources/mws_storage_configurations.md) | No |
+| [databricks_mws_workspaces](docs/resources/mws_workspaces.md) | No |
+| [databricks_node_type](docs/data-sources/node_type.md) data | No |
+| [databricks_notebook](docs/resources/notebook.md) | Yes |
+| [databricks_notebook](docs/data-sources/notebook.md) data | Yes |
+| [databricks_notebook_paths](docs/data-sources/notebook_paths.md) data | No |
+| [databricks_obo_token](docs/resources/obo_token.md) | No |
+| [databricks_permissions (job)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (cluster)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (instance_pool)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (cluster_policy)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (repo)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (token)](docs/resources/permissions.md) | No |
+| [databricks_permissions (password)](docs/resources/permissions.md) | No |
+| [databricks_permissions (delta live tables pipeline)](docs/resources/permissions.md) | No |
+| [databricks_permissions (notebook)](docs/resources/permissions.md) | No |
+| [databricks_permissions (directory)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow experiment)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow registered model)](docs/resources/permissions.md) | No |
+| [databricks_permissions (sql_endpoint) ](docs/resources/permissions.md) | No |
+| [databricks_pipeline](docs/resources/pipeline.md) | No |
+| [databricks_repo](docs/resources/repo.md) | Yes |
+| [databricks_schema](docs/resources/schema.md) | No |
+| [databricks_schemas](docs/data-sources/schema.md) data | No |
+| [databricks_secret](docs/resources/secret.md) | Yes |
+| [databricks_secret_acl](docs/resources/secret_acl.md) | Yes |
+| [databricks_secret_scope](docs/resources/secret_scope.md) | Yes |
+| [databricks_spark_version](docs/data-sources/spark_version.md) data | No |
```

Comment:
data sources doesn't make sense to expose. I recommend to leave only resources in this list